its/src/test/java/com/sonar/it/csharp/TestProjectTest.java

```diff
@@ -94,4 +97,21 @@ public void with_html_and_csharp_code_explicitly_marked_as_test_should_not_popul
     assertThat(buildResult.getLogsLines(l -> l.contains("INFO"))).contains("INFO: Found 1 MSBuild C# project: 1 TEST project.");
     TestUtils.verifyGuiTestOnlyProjectAnalysisWarning(ORCHESTRATOR, buildResult, "C#");
   }
+
+  @Test
```

Comment:
could you move this above `with_html_and_csharp_code_explicitly_marked_as_test_should_not_populate_metrics` ? to have all CSHARP_ONLY_TEST_PROJECT tests grouped together